js-setup/templates/bootstrapper.html

```diff
@@ -54,15 +54,15 @@
 				'Array.prototype.find|always|gated',
 				'Array.prototype.findIndex|always|gated',
 				'IntersectionObserver'
-			].join(',') + '&excludes=Symbol,Symbol.iterator,Symbol.species,Map,Set');
+			].join(',') + '&excludes=Symbol,Symbol.iterator,Symbol.species,Map,Set' + '&flags=gated&unknown=polyfill');
 		} else {
 			{{#if hasMainCoreJs}}
 			ftNextLoadScript('{{#hashedAsset}}main-core.js{{/hashedAsset}}');
 			{{/if}}
 			ftNextLoadScript(polyfillUrl + '?callback=' + polyfillCallbackName + '&features=' + [
 				'default',
 				'HTMLPictureElement|always|gated'
-			].join(','));
+			].join(',') + '&flags=gated&unknown=polyfill');
```

Comment:
@JakeChampion @quarterto does this look good to you? What testing should we do before and after? Apart from JS not working when links opened within Facebook app? Thank-you very much :)